pkg/notifications/notifications_api.go

```diff
@@ -184,6 +184,12 @@ const AiNotificationsCreateDestinationMutation = `mutation(
 			  authType
 			  prefix
 			}
+			... on AiNotificationsCustomHeadersAuth {
+			  authType
+        	  customHeaders {
+          	    key
+			  }
+			}
```

Comment:
<img width="460" alt="image" src="https://github.com/newrelic/newrelic-client-go/assets/127438038/3324dfde-fe69-4b92-a9ea-4d5223e65e84">

@nzur-newrelic do we also need to add the attribute `value` here?

Reply:
Apologies - my bad; I mistook this to be the attributes in the response

Reply:
No, the part from the screenshot is the parameter, while in the code you're looking at is the response.

Reply:
@pranav-new-relic